README.md

```diff
@@ -38,6 +38,31 @@ cluster on AWS.
 
 ------
 
+## Compatibility with Cluster API and Kubernetes Versions
+
+This provider's versions are compatible with the following versions of Cluster API:
+
+||Cluster API v1alpha1 (v0.1)|
+|-|-|
+|AWS Provider 0.2|✓|
```

Comment:
Not sure I understand what `AWS Provider 0.2` means here.
Can you please clarify?

Reply:
This is meant to be what we release on 3/29.

Reply:
Then @detiber's suggestion makes that more clear.
Also, suggest making it clear that this is an `alpha` release will be helpful. WDYT?

Reply:
Which do you like best to follow `AWS Provider`?
- v1alpha1 (v0.2)
- v0.2 (alpha)
- v1alpha1 (v0.2)
- ...

Reply:
`v1alpha1 (v0.2)`

Reply:
Made suggestions to save you the copy editing :)

Reply:
eh, i'd already made the edits locally, but thanks!